lib/arelastic/aggregations/children.rb

```diff
@@ -0,0 +1,9 @@
+module Arelastic
+  module Aggregations
+    class Children < Arelastic::Aggregations::SpecialSingleBucket
```

Comment:
How about `Association` or `Associated` instead of `Special`?

Reply:
`Special` is the terminology used in the [docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-parent-aggregation.html) to describe these types of aggregations.